test/bdd/pkg/context/context.go

```diff
@@ -40,7 +40,7 @@ func NewBDDContext(caCertPath, testDataPath string) (*BDDContext, error) {
 		return nil, err
 	}
 
-	vdri, err := createVDRI("http://localhost:8080/1.0/identifiers")
+	vdri, err := createVDRI("http://localhost:8080/1.0/identifiers", "http://localhost:8072/1.0/identifiers")
```

Comment:
some constants would make that more clear :).

Reply:
[Done](https://github.com/trustbloc/edge-service/compare/20d35c6e19eab6192e155c79da794478da86e291..4585fcb01eb809caf81f16aa0068926c2de077f0).